web/package.json

```diff
@@ -228,4 +132,4 @@
     "@types/react": "18.3.5",
     "webpack": "5.94.0"
   }
-}
+}
```

Comment:
I think removing this space here creates working changes when you `yarn install`